From caacab095f946fffdd938edb871f4a80d3701430 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Fri, 3 Jun 2016 15:44:23 +0200
Subject: [PATCH 26/26] S Core: Activate score networking

Ticket #931
---
 include/configs/mx6sxscore.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mx6sxscore.h b/include/configs/mx6sxscore.h
index 84dadde..89ad3a4 100644
--- a/include/configs/mx6sxscore.h
+++ b/include/configs/mx6sxscore.h
@@ -69,6 +69,7 @@
 #undef CONFIG_BOOTM_PLAN9
 #undef CONFIG_BOOTM_RTEMS
 
+#define CONFIG_SCORE_NETWORK
 #ifdef CONFIG_SCORE_NETWORK
 /* Network configs */
 #define CONFIG_CMD_PING
-- 
1.9.1

