From 423f65e777f5756ce4719f6c919f26d2d07e8524 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Mon, 3 Aug 2015 10:08:23 +0200
Subject: [PATCH] Makefile: Add sitec label to EXTRAVERSION

Signed-off-by: Robert Lehmann <robert.lehmann@sitec-systems.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a2e572b..f9a7624 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 3
 PATCHLEVEL = 14
 SUBLEVEL = 28
-EXTRAVERSION =
+EXTRAVERSION = -sitec
 NAME = Remembering Coco
 
 # *DOCUMENTATION*
-- 
1.9.1

